bosh/releases/pre_render_scripts/router/gorouter/jobs/patch_pre-start.sh

```diff
@@ -10,13 +10,18 @@ if [[ -f "${sentinel}" ]]; then
 fi
 
 patch --verbose "${target}" <<'EOT'
-@@ -27,6 +27,3 @@
- <% else %>
+--- pre-start.erb  2019-12-12 13:56:59.944789605 +0100
++++ -  2019-12-12 13:57:07.384803023 +0100
+@@ -28,9 +28,6 @@
      echo "Not setting /proc/sys/net/ipv4 parameters, since I'm running inside a linux container"
  <% end %>
--
+
 -# Allowed number of open file descriptors
 -ulimit -n 100000
+-
+ # Add jq to path
```

Comment:
could you also say "why" in the comment please?

Reply:
Oh, this might be a misconception. I didn't add jq, I only adjusted the patch to apply cleanly again (see  https://github.com/cloudfoundry/routing-release/blob/develop/jobs/gorouter/templates/pre-start.erb#L34).